pynamodb_attributes/unicode_datetime.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from datetime import timezone
+from typing import Any
+from typing import Optional
+
+import pynamodb.attributes
+from pynamodb.attributes import Attribute
+
+
+class UnicodeDatetimeAttribute(Attribute[datetime]):
+    """
+    Stores a 'datetime.datetime' object as an ISO8601 formatted string
+
+    This is useful for wanting database readable datetime objects that also sort.
+
+    >>> class MyModel(Model):
+    >>>   created_at = UnicodeDatetimeAttribute()
+    """
+
+    attr_type = pynamodb.attributes.STRING
+
+    def __init__(
+        self,
```

Comment:
Maybe make those kwargs-only?
```suggestion
        self,
        *,
```